pkg/config/config.go

```diff
@@ -228,7 +228,7 @@ type Blockstore struct {
 }
 
 func (b *Blockstore) GetStorageIDs() []string {
-	return nil
+	return []string{""}
```

Comment:
```suggestion
	return []string{SingleBlockstoreID}
```

Reply:
And while we're at it, `GetStorageByID` should also be used in `GetStorageByID` below.